### DISCUSSION TITLE
How can I use Split conditionally when device is md?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#14138] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

How can I use Split conditionally by hiding it when device is md?  There is only a `->from()` breakpoint to use which means I can't hide it from md+ devices. I want to use normal table with headers and columns on md+ devices and Split and Stack on small devices.  This seems logical.  How do we do it?  

Also, why can't we use Stack in normal tables?  Looks like it only works within Split.

**Comment by e2d2-dev** [comment]
hi,

there is ->visibleFrom() and ->hiddenFrom()

**Reply by myregistration** [reply]
Yeah, I tried that thanks!  I think you need to use Split, even with Stack, but then you lose the table headers.  I want to display a normal table with headers when on a non-mobile device.  And I believe visibleFrom and hiddenFrom work with Stack, but not Split.  I need to confirm.  I ended up using Livewire to save the device size so I can create 2 different layouts for Tables.  I figured that might be helpful for other circumstances as well.

**Comment by yoshi315** [comment]
Hi, you might need install https://github.com/jenssegers/agent first, then detect the browser device, and then decide whether use Split or not.

I choose this way in my own projects.
